components/scream/src/physics/p3/p3_constants.hpp

```diff
@@ -29,24 +29,39 @@ struct Constants
   static constexpr Scalar Pi          = 3.14159265;
   static constexpr long long int    iulog       = 98;
   static constexpr bool   masterproc  = true;
-  static constexpr Scalar RHOW     = RhoH2O;
-  static constexpr Scalar INV_RHOW = 1.0/RHOW;
-  static constexpr Scalar THIRD    = 1.0/3.0;
-  static constexpr Scalar SXTH     = 1.0/6.0;
-  static constexpr Scalar PIOV6    = Pi*SXTH;
-  static constexpr Scalar CONS1    = PIOV6*RHOW;
-  static constexpr Scalar QSMALL   = 1.e-14;
-  static constexpr Scalar NSMALL   = 1.e-16;
-  static constexpr Scalar P0       = 100000.0;        // reference pressure, Pa
-  static constexpr Scalar RD       = 287.15;          // gas constant for dry air, J/kg/K
-  static constexpr Scalar RHOSUR   = P0/(RD*273.15);
-  static constexpr Scalar CP       = Cpair;          // heat constant of air at constant pressure, J/kg
-  static constexpr Scalar INV_CP   = 1.0/CP;
+  static constexpr Scalar RHOW        = RhoH2O;
+  static constexpr Scalar INV_RHOW    = 1.0/RHOW;
+  static constexpr Scalar THIRD       = 1.0/3.0;
+  static constexpr Scalar SXTH        = 1.0/6.0;
+  static constexpr Scalar PIOV6       = Pi*SXTH;
+  static constexpr Scalar CONS1       = PIOV6*RHOW;
+  static constexpr Scalar QSMALL      = 1.e-14;
+  static constexpr Scalar NSMALL      = 1.e-16;
+  static constexpr Scalar P0          = 100000.0;        // reference pressure, Pa
+  static constexpr Scalar RD          = 287.15;          // gas constant for dry air, J/kg/K
+  static constexpr Scalar RHOSUR      = P0/(RD*273.15);
+  static constexpr Scalar CP          = Cpair;          // heat constant of air at constant pressure, J/kg
+  static constexpr Scalar INV_CP      = 1.0/CP;
+
+  // Constants for ice lookup tables
+  static constexpr int    DENSIZE     = 5;
```

Comment:
Not sure if these belong here. In fortran, they come from micro_p3_utils.

Reply:
These are universal constants that should be pulled from a central location for all of scream when the time comes.  We haven't set that up yet in the SCREAM infrastructure but we will want to pull these values from there once we do.  

Side note, micro_p3_utils serves as a middle-man for these values which are taken from a module that is in CIME that contains universal constants.